src/include/platform/DeviceControlServer.h

```diff
@@ -121,8 +141,7 @@ class DeviceControlServer final
     DeviceControlServer(const DeviceControlServer &&) = delete;
     DeviceControlServer & operator=(const DeviceControlServer &) = delete;
 
-    NodeId mPeerNodeId  = 0;
-    FabricIndex mFabric = 0;
+    NodeId mPeerNodeId = kUndefinedNodeId;
```

Comment:
While you're here: is mPeerNodeId used?  Can we just remove it?  Separate PR ok.

Reply:
Only SetPeerNodeId is used by by general-commisioning cluster, https://github.com/project-chip/connectedhomeip/issues/15366